prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -442,7 +443,11 @@ private boolean checkEntityInstanceAvailability(String entityName, String cluste
                                                     String entityType) throws
         FalconException {
         Entity entity = EntityUtil.getEntity(entityType, entityName);
-
+        if (entity.getACL().getOwner() != null && !entity.getACL().getOwner().isEmpty()) {
```

Comment:
StringUtils.isNotBlank() can be used, instead of checking for null and empty condition for owner String value.